pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -77,9 +78,14 @@ func (a *SingleNodeBootstrapInPlace) Generate(dependencies asset.Parents) error
 	}
 
 	a.Config = a.Bootstrap.Config
+	data, err := ignition.Marshal(a.Config)
+	if err != nil {
+		return errors.Wrap(err, "failed to Marshal Ignition config")
+	}
+
 	a.File = &asset.File{
 		Filename: singleNodeBootstrapInPlaceIgnFilename,
-		Data:     a.Bootstrap.File.Data,
```

Comment:
We have a problem here with consistency in the `Bootstrap` field between `Generate` and `Load`. This will get fixed up with my PR to introduce a common bootstrap type.